.github/workflows/repository-github-super-linter.yml

```diff
@@ -2,39 +2,62 @@
 name: GitHub Super-Linter
 
 on: # yamllint disable-line rule:truthy
-  pull_request:
-    branches:
-      - main
-    types:
-      - edited
-      - opened
-      - reopened
-      - synchronize
-
   schedule:
     - cron: 17 4 * * 1-5
 
-permissions: read-all
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  group: '${{ github.ref }}-${{ github.workflow }}'
+  cancel-in-progress: true
 
 jobs:
-  super-linter:
+  build:
     name: GitHub Super-Linter
     runs-on: ubuntu-latest
-    permissions:
-      statuses: write
     steps:
-      - name: Checkout
-        id: checkout
-        uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+      - name: Checkout Code
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab #v3.5.2
         with:
+          token: '${{ secrets.GITHUB_TOKEN }}'
           fetch-depth: 0
 
+      - name: branch name
+        id: name
+        run: |
+          date=$(date +%Y_%m_%d)
+          branch_name="date_$date"
+          echo "::set-output name=value::$branch_name"
+
+      - name: Check branch
+        id: branch
+        run: |
+          if git ls-remote --exit-code --heads origin ${{ steps.name.outputs.value }}; then
+            echo "$branch_name branch exists"
+            echo "::set-output name=exists::true"
+            exit 0
+          fi
+          echo "::set-output name=exists::false"
+           
+      - name: Create new branch
+        if: steps.branch.outputs.exists == 'false'
+        run: |
+          git checkout -b ${{ steps.name.outputs.value }}
+          git push -u origin ${{ steps.name.outputs.value }}
+        
+      - name: checkout branch
+        if: steps.branch.outputs.exists == 'true'
+        run: |
+          git checkout ${{ steps.name.outputs.value }}
+
       - name: GitHub Super-Linter
         id: super_linter
         uses: github/super-linter/slim@454ba4482ce2cd0c505bc592e83c06e1e37ade61 # v4.10.1
```

Comment:
Super-Linter is now `super-linter/super-linter` and is currently at [v5.0.0](https://github.com/super-linter/super-linter/releases/tag/v5.0.0)